app/scripts/locales/base.json

```diff
@@ -428,6 +429,8 @@
   "setFileDontUseKeyFile": "Don't use key file",
   "setFileEmptyPass": "Empty password",
   "setFileEmptyPassBody": "Saving database with an empty password makes it completely unprotected. Do you really want to do it?",
+  "setFilePasswordError": "Error updating password",
+  "setFilePasswordErrorBody": "Master password does not match the confirm master password.",
```

Comment:
"confirm master password" should be probably something else, like "confirmed password", or something that make a sentence